app/helpers/hyku_addons/creator_field_helper.rb

```diff
@@ -7,15 +8,15 @@ def add_pacific_creator_personal_fields(array)
     end
 
     def add_redlands_creator_personal_fields(array, service_options)
-      array.delete_at(6) # removes institutional relationship from Redlands worktype
+      array.delete_at(7) # removes institutional relationship from Redlands worktype
```

Comment:
Is this from your PR, or is this an outdated change?

Reply:
This is due to the addition of the `creator_instiutional_email` field to the array